docs/content/Coding Milestones/PoC2023q1/example1-subs/example1-stage-1a.md

```diff
@@ -83,24 +80,22 @@ espw                                                       <none>       2n88ugkh
 imw1                                                       <none>       4d2r9stcyy2qq5c1
 ```
 
-Also: if you ever need to look up just one mailbox workspace by
+Also: if you ever need to look up just one mailbox space by
 SyncTarget name, you could do it as follows.
 
 ```shell
-# TODO: currently some workspaces are not created as spaces, specifically the mailbox workspaces, so leaving this code
-GUILDER_WS=$(kubectl get Workspace -o json | jq -r '.items | .[] | .metadata | select(.annotations ["edge.kubestellar.io/sync-target-name"] == "guilder") | .name')
-echo The guilder mailbox workspace name is $GUILDER_WS
+GUILDER_SPACE=$(KUBECONFIG=$SM_CONFIG kubectl get space -n spaceprovider-default -o json | jq -r '.items | .[] | .metadata | select(.annotations ["edge.kubestellar.io/sync-target-name"] == "guilder") | .name')
+echo The guilder mailbox space name is $GUILDER_SPACE
 ```
 ``` { .bash .no-copy }
-The guilder mailbox workspace name is 1t82bk54r6gjnzsp-mb-f0a82ab1-63f4-49ea-954d-3a41a35a9f1c
+The guilder mailbox space name is 1t82bk54r6gjnzsp-mb-f0a82ab1-63f4-49ea-954d-3a41a35a9f1c
 ```
 
 ```shell
-# TODO: currently some workspaces are not created as spaces, specifically the mailbox workspaces, so leaving this code
-FLORIN_WS=$(kubectl get Workspace -o json | jq -r '.items | .[] | .metadata | select(.annotations ["edge.kubestellar.io/sync-target-name"] == "florin") | .name')
-echo The florin mailbox workspace name is $FLORIN_WS
+FLORIN_SPACE=$(KUBECONFIG=$SM_CONFIG kubectl get space -n spaceprovider-default -o json | jq -r '.items | .[] | .metadata | select(.annotations ["edge.kubestellar.io/sync-target-name"] == "florin") | .name')
```

Comment:
This variable name is misleading. It is not about the florin workload execution cluster (remember a cluster is a space). It is about florin's _mailbox_ space. So I would name it "FLORIN_MB_SPACE".